views/components/tabular-data.html

```diff
@@ -0,0 +1,29 @@
+{% macro tabularData(myLatest) %}
+<article class="base--article" itemscope itemtype="http://schema.org/Table">
+  <header class="base--header">
+    <h1 class="base--h1" itemprop="name">{{myLatest.title}}</h1>
+  </header>
+  <table class="base--table">
+    {% if myLatest.caption %}
+    <caption class="base--caption" itemprop="caption" itemscope itemtype="https://schema.org/Property">{{myLatest.caption}}</caption>
+    {% endif %}
+    <thead class="base--thead">
+      <tr class="base--tr">
+        {% for tableHeader in myLatest.tableHeaders %}
+        <th class="base--th">{{tableHeader}}</th>
+        {% endfor %}
+      </tr>
+    </thead>
+    <tbody class="base--tbody">
+        {% for table in myLatest.table %}
+        <tr class="base--tr">
+          <td data-th="{{table.headers[0]}}" class="base--td" itemprop="name" itemscope itemtype="https://schema.org/Table">{{table.identifier}}</td>
```

Comment:
You seem to be duplicating header information here. The `data-th` should be drawing from the same item generating the `th`s (`data.headers`, as I'm recommending). This also hardcodes the number table data cells in a row, regardless of number of headers. The number of cells should be either dynamically generated based on the number of headers there are, or it should be dynamically generated based on an array of values.

I'd recommend something like the following:

```html
{% for row in data.body %}
<tr class="base--tr">
  {% for header in data.headers %}
  <td data-th="{{header}}" class="base--td">{{row[loop.index]}}</td>
  {% endfor %}
</tr>
{% endfor %}
```

`data.body` should be an array of values